File: ReactApp/address-app-react/src/app/models/addresses/AddressSearchModel.ts
import { IAddressSearchModel } from "./interfaces/IAddressSearchModel";

export class AddressSearchModel implements IAddressSearchModel{
    id: string | null = null;
    streetNumber: string | null =null;
    streetName: string | null = null;
    city: string | null = null;
    stateId: string | null = null;
    zipCode: string | null = null;

}

File: BlazorApp/BlazorAddressApp/BlazorAddressApp/AddressApp/Models/Addresses/AddressSearchModel.cs
namespace AddressApp.Models.Locations
{
    public class AddressSearchModel
    {
        public string StreetNumber { get; set; }
        public string StreetName { get; set; }
        public string ApartmentNumber { get; set; }
        public string City { get; set; }
        public Guid? StateId { get; set; }
        public string ZipCode { get; set; }

    }
}
